src/interface/obsidian/src/chat_modal.ts

```diff
@@ -328,60 +393,17 @@ export class KhojChatModal extends Modal {
                     }
                 } catch (error) {
                     // If the chunk is not a JSON object, just display it as is
```

Comment:
Is this comment accurate? Or is this just for displaying the error?

Reply:
Yeah, this error is catches failure to parse the `response` as JSON